src/java/org/apache/nutch/crawl/DeduplicationJob.java

```diff
@@ -31,7 +31,7 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.io.BytesWritable;
 import org.apache.hadoop.io.Text;
-import org.apache.hadoop.mapred.Counters.Group;
+/*import org.apache.hadoop.mapred.Counters.Group;
```

Comment:
Never leave code commented out like this please. Also make explicit individual imports please.